.github/workflows/deployment_tests.yml

```diff
@@ -1,237 +0,0 @@
-name: 'Deployment Tests'
-
-on:
-  push:
-    branches:
-      - main
-      - develop
-
-jobs:
-  ############################# Deployment Tests for MCP modules #####################################################
-  MCP-tests:
```

Comment:
The MCP-tests job is still needed. This job creates a google project where a Cloud Run service, App Engine App and Kubernetes Cluster are deployed for testing deployment with the MCP modules